tripgo.swagger.yaml

```diff
@@ -90,6 +90,110 @@ paths:
           schema:
             $ref: '#/definitions/RegionInfoResponse'
 
+  /regionDetails.json:
+    post:
+      tags:
+        - Configuration
+      summary: Detailed TSPs per region
+      description:
+        Retrieves detailed information about covered transport service providers for a specified region.
+      parameters:
+        - name: input
+          in: body
+          schema:
+            type: object
+            properties:
+              region:
+                type: string
+                description: Region name/code from `regions.json`
+              modes:
+                type: array
+                items:
+                  type: string
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              operators:
+                type: array
+                items:
+                  type: string
+                description: TSPs name for which results should be returned. In absence, all operators are included.
```

Comment:
should we make it an error?  if they want empty, they should ask for empty.  if they try an invalid combination, can we say `operator found in region, but does not match selected modes`.